### PR TITLE
Fix redis-ha README.md with correct server values param

### PR DESCRIPTION
**Pull request by victortrac**
https://github.com/kubernetes/charts/blob/master/stable/redis-ha/templates/redis-server-deployment.yaml#L34-L35 contains this:

```yaml
        resources:
{{ toYaml .Values.resources.server | indent 10 }}
```

The README.md refers to keys for master/slave that aren't accurate.

### CODE REVIEW

**Review by carrodher — CHANGES_REQUESTED** [review]
You should increment chart version (`version`) in `Chart.yaml`

File: stable/redis-ha/README.md [hunk 1]
```diff
@@ -61,8 +61,7 @@ The following table lists the configurable parameters of the Redis chart and the
...
 | -------------------------------- | -----------------------------------------------------                                                                        | --------------------------------------------------------- |
 | `redis_image`                    | Redis image                                                                                                                  | `quay.io/smile/redis:4.0.6r2`                             |
-| `resources.master`               | CPU/Memory for master nodes resource requests/limits                                                                         | Memory: `200Mi`, CPU: `100m`                              |
-| `resources.slave`                | CPU/Memory for slave nodes  resource requests/limits                                                                         | Memory: `200Mi`, CPU: `100m`                              |
+| `resources.server`               | CPU/Memory for master/slave nodes resource requests/limits                                                                   | Memory: `200Mi`, CPU: `100m`                              |
```
Comment (timstoop) [inline stable/redis-ha/README.md:64]: All changes, however small, require a bump in the Charts.yaml `version`. For this, you can just use a patch number increase.

**Review by timstoop — CHANGES_REQUESTED** [review]

**Pull request merged**